labot/protocolBuilder.py

```diff
@@ -3,9 +3,11 @@
 import pickle
 
 from pprint import pprint
+from typing import Union
 
 class_pattern =\
-    r"\s*public class (?P<name>\w+) (?:extends (?P<parent>\w+) )?implements (?P<interface>\w+)\n"
+    r"\s*public class (?P<name>\w+) (?:extends (?P<parent>\w+) )?implements (?P<interface>\w+(?:, \w+)?)\n"
```

Comment:
Yes, here: https://regex101.com/r/X1J3nM/10

Your regex failed on `NetworkDataContainerMessage`s because it implements two interfaces (apparently it's the only kind of message to do so).